elasticdl/python/master/master.py

```diff
@@ -127,6 +147,18 @@ def __init__(self, args):
         self._should_stop = False
         self._exit_code = 0
 
+    def _set_completed_steps_by_checkpoint(self, checkpoint_dir_for_init):
+        if not checkpoint_dir_for_init:
+            return
+
+        CheckpointSaver.check_checkpoint_valid(checkpoint_dir_for_init)
```

Comment:
If check_checkpoint_valid returns false, we need return directly instead of executing the following logic

Reply:
Yes, it may be better to raise an error if checkpoint dir is not None and invalid.